CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 0.102.2
+
```

Comment:
```suggestion

*  Fix passing BZs when running packit `propose-downstream`. (#2447)

```

@nforro is this ok to mention this like this?

Reply:
```suggestion

*  Fixed passing list of resolved bugs when running `packit propose-downstream` or `packit pull-from-upstream`. (#2447)

```